src/core/DataFile.cpp

```diff
@@ -868,10 +869,21 @@ void DataFile::upgrade_1_1_91()
 	for( int i = 0; !list.item( i ).isNull(); ++i )
 	{
 		QDomElement el = list.item( i ).toElement();
-		if ( el.attribute( "name" ) == "plugin" && el.attribute( "value" ) == "vocoder-lmms" ) {
+		if( el.attribute( "name" ) == "plugin" && el.attribute( "value" ) == "vocoder-lmms" ) {
 			el.setAttribute( "value", "vocoder" );
 		}
 	}
+
+	list = elementsByTagName( "crossoevereqcontrols" );
+	for( int i = 0; !list.item( i ).isNull(); ++i )
+	{
+		QDomElement el = list.item( i ).toElement();
+		// invert the mute LEDs
+		for( int d = 1; d <= 4; ++d ){
```

Comment:
> there is no way the older version has more than 4 LED's, and this is for upgrading old projects only

Good point.  My logic was off.  In that case, it can stay; up to you.